erpnext/accounts/doctype/financial_report_template/financial_report_template.py

```
@@ -0,0 +1,182 @@
+
+	from typing import TYPE_CHECKING
+
+	if TYPE_CHECKING:
+		from frappe.types import DF
+
+		from erpnext.accounts.doctype.financial_report_row.financial_report_row import FinancialReportRow
+
+		disabled: DF.Check
+		module: DF.Link | None
+		report_type: DF.Literal[
+			"", "Profit and Loss Statement", "Balance Sheet", "Cash Flow", "Custom Financial Statement"
```

Ruff (0.14.1)

Syntax error in forward annotation: Expected an expression

(F722)